pkg/manager/member/startscript/v1/template.go

```diff
@@ -105,10 +109,12 @@ exec /tidb-server ${ARGS}
 type TidbStartScriptModel struct {
 	CommonModel
 
-	EnablePlugin    bool
-	PluginDirectory string
-	PluginList      string
-	Path            string
+	EnablePlugin       bool
+	PluginDirectory    string
+	PluginList         string
+	Path               string
+	EnableBootstrapSQL bool
+	BootstrapSQLFile   string
```

Comment:
How about just using `BootstrapSQLFile *string`, and no need to use `EnableBootstrapSQL`?